<?xml version="1.0"?>

<launch>	
	<!-- Odom launch -->
	<node pkg="robocare_frame" type="robocare_frame" name="robocare_frame"/>
	
	<!-- laser launch-->	
	<node pkg="tf" type="static_transform_publisher" name="hukuyo_broadcaster" args="0.12 0 0 0 0 0 1 base_link laser 100" />
	<node name="urg_node" pkg="urg_node" type="urg_node"/>		
</launch>
